src/Numerics.Tests/RootFindingTests/FindRootsTest.cs

```diff
@@ -210,5 +211,113 @@ public void QuadraticExpected(double c, double b, double a, double x1R, double x
             AssertComplexEqual(Complex.Zero, c + b*x1 + a*x1*x1, 1e-14);
             AssertComplexEqual(Complex.Zero, c + b*x2 + a*x2*x2, 1e-14);
         }
+
+        [Test]
+        public void RootsTestEmpty()
```

Comment:
I would keep the naming convention as it currently is, i.e. no 'Test' in the function name (even though I don't agree with this convention)